apps/backend/src/app/modules/admin/admin.controller.ts

```diff
@@ -275,7 +296,65 @@ export class AdminController {
     description: 'The XP System variables to set',
     required: true
   })
-  updateXPSystems(@Body() body: UpdateXpSystemsDto) {
-    return this.xpSystems.update(body);
+  updateXPSystems(
+    @LoggedInUser('id') adminID: number,
+    @Body() body: UpdateXpSystemsDto
+  ) {
+    return this.xpSystems.update(body, adminID);
+  }
+
+  @Get('/activities')
+  @Roles(RolesEnum.ADMIN, RolesEnum.MODERATOR)
+  @ApiOperation({
+    description: 'Get list of admin activities'
+  })
+  @ApiOkResponse({
+    type: AdminActivityDto,
+    description: 'List of admin activities'
+  })
+  @ApiBadRequestResponse({
+    description: 'Requested user is neither admin nor moderator'
+  })
+  @ApiForbiddenResponse({
+    description: "Moderators can't get admin's activities"
+  })
+  getSelfAdminActivities(
+    @LoggedInUser('id') adminID: number,
+    @Query() query: AdminGetAdminActivitiesQueryDto
+  ): Promise<PagedResponseDto<AdminActivityDto>> {
+    return this.adminActivityService.getList(
+      adminID,
+      null,
+      query.skip,
+      query.take
+    );
+  }
+
+  @Get('/activities/:userID')
+  @Roles(RolesEnum.ADMIN, RolesEnum.MODERATOR)
+  @ApiOperation({
+    description: 'Get list of admin activities'
+  })
+  @ApiOkResponse({
+    type: AdminActivityDto,
+    description: 'List of admin activities'
+  })
+  @ApiBadRequestResponse({
+    description: 'Requested user is neither admin nor moderator'
+  })
+  @ApiForbiddenResponse({
+    description: "Moderators can't get admin's activities"
+  })
+  getAdminActivities(
+    @LoggedInUser('id') adminID: number,
+    @Param('userID', ParseIntSafePipe) userID: number,
+    @Query() query: AdminGetAdminActivitiesQueryDto
+  ): Promise<PagedResponseDto<AdminActivityDto>> {
+    return this.adminActivityService.getList(
+      adminID,
+      userID,
+      query.skip,
+      query.take
+    );
   }
```

Comment:
I think this should show all admin/mod activities by default, and take an optional `userID` parameter.

I realise that makes restricting admin activities from mods a lot harder, but struggling to think of scenarios where admin actions need to be hidden. Sorry, I realise I said we should restrict them in the issue for this feature, thinking about it it seems like the wrong approach. In practice we want this to be a way for all mods and admins to stay in the loop of what's going on throughout the game, so a page showing *all* activities would be really nice. We have a #moderators channel in our Discord where we manually document doc actions (because Discord's audit log is a bit crap), and it's really really annoying.